WorldModel/WorldModel/Core/Core.aslx

```diff
@@ -126,6 +126,15 @@
       else {
         JS.setCss ("#status", "border:none")
       }
+      if (HasString(game, "locationbarimage")){
+        url = game.locationbarimage
+        if (not Trim(url) = ""){ 
+          if (not StartsWith(url,  "http://") and not StartsWith(url, "https://")){
+            url = GetFileURL(url)
+            JS.eval ("$('.ui-widget-header').css('background-image','url(" + url + ")');")
+          }
+        }

```

Comment:
`url` isn't an accurate variable name here outside the `if`. Also, I guess we don't actually need the `http://` check? Not sure what happens if you pass a URL into `GetFileURL`... either way I presume we don't do a check like this anywhere else the author can supply images.

```suggestion
        if (not Trim(game.locationbarimage) = ""){ 
          url = GetFileURL(game.locationbarimage)
          JS.eval ("$('.ui-widget-header').css('background-image','url(" + url + ")');")
        }
```